api/src/main/java/com/codeforcommunity/dto/site/UpdateSiteRequest.java

```diff
@@ -0,0 +1,440 @@
+  private Boolean pooling;
+  private Boolean stakes_with_wires;
+  private Boolean stakes_without_wires;
+  private Boolean light;
+  private Boolean bicycle;
+  private Boolean bag_empty;
+  private Boolean bag_filled;
+  private Boolean tape;
+  private Boolean sucker_growth;
+  private String site_type;
+  private String sidewalk_width;
+  private Double site_width;
+  private Double site_length;
+  private String material;
+  private Boolean raised_bed;
+  private Boolean fence;
+  private Boolean trash;
+  private Boolean wires;
+  private Boolean grate;
+  private Boolean stump;
+  private String tree_notes;
+  private String site_notes;
+
+  public UpdateSiteRequest(
+      Boolean tree_present,
+      String status,
+      String genus,
+      String species,
+      String common_name,
+      String confidence,
+      Double diameter,
+      Double circumference,
+      Boolean multistem,
+      String coverage,
+      String pruning,
+      String condition,
+      Boolean discoloring,
+      Boolean leaning,
+      Boolean constricting_gate,
+      Boolean wounds,
+      Boolean pooling,
+      Boolean stakes_with_wires,
+      Boolean stakes_without_wires,
+      Boolean light,
+      Boolean bicycle,
+      Boolean bag_empty,
+      Boolean bag_filled,
+      Boolean tape,
+      Boolean sucker_growth,
+      String site_type,
+      String sidewalk_width,
+      Double site_width,
+      Double site_length,
+      String material,
+      Boolean raised_bed,
+      Boolean fence,
+      Boolean trash,
+      Boolean wires,
+      Boolean grate,
+      Boolean stump,
+      String tree_notes,
+      String site_notes) {
+    this.tree_present = tree_present;
+    this.status = status;
+    this.genus = genus;
+    this.species = species;
+    this.common_name = common_name;
+    this.confidence = confidence;
+    this.diameter = diameter;
+    this.circumference = circumference;
+    this.multistem = multistem;
+    this.coverage = coverage;
+    this.pruning = pruning;
+    this.condition = condition;
+    this.discoloring = discoloring;
+    this.leaning = leaning;
+    this.constricting_gate = constricting_gate;
+    this.wounds = wounds;
+    this.pooling = pooling;
+    this.stakes_with_wires = stakes_with_wires;
+    this.stakes_without_wires = stakes_without_wires;
+    this.light = light;
+    this.bicycle = bicycle;
+    this.bag_empty = bag_empty;
+    this.bag_filled = bag_filled;
+    this.tape = tape;
+    this.sucker_growth = sucker_growth;
+    this.site_type = site_type;
+    this.sidewalk_width = sidewalk_width;
+    this.site_width = site_width;
+    this.site_length = site_length;
+    this.material = material;
+    this.raised_bed = raised_bed;
+    this.fence = fence;
+    this.trash = trash;
+    this.wires = wires;
+    this.grate = grate;
+    this.stump = stump;
+    this.tree_notes = tree_notes;
+    this.site_notes = site_notes;
+  }
+
+  private UpdateSiteRequest() {}
+
+  public Boolean isTree_present() {
```

Comment:
Do we want this underscore notation here? We should probably use camel case unless this messes with how its written to JSON